# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libwmf/hotfix-include.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libwmf-0.2.17/src/ipa/ipa/bmp.h.vanilla	2026-10-30 09:36:46.028797711 +0200
+++ libwmf-0.2.17/src/ipa/ipa/bmp.h	2026-10-30 09:37:19.787669803 +0200
@@ -16,6 +16,7 @@
    write to the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
    Boston, MA 02111-1307, USA.  */
 
+#include <stdint.h>
 
 #ifndef HAVE_GD
 /* png stuff below adapted from stuff sent to me [fjf] by Lennard D. Rosenthal;
